tests/test_board.py

```diff
@@ -0,0 +1,53 @@
+from context import src
+from src import board as Board
+from src import cell as Cell
+import unittest
+
+
+class TestBoardProperties(unittest.TestCase):
+
+    def test_create_empty_board(self):
+        empty_board = Board.Board(0, 0)
+        self.assertEqual(len(empty_board._board), 0)
+
+    def test_create_small_board(self):
+        small_board = Board.Board(2, 3)
```

Comment:
I don't know if you cannot do this with unittest, but in general if we have an object we want to use for our test it would be better to define it once outside the test function at the class level. Seems to be the case here. I think in pytest youhave a setup method where you can setup all the objects you need

Reply:
Done, defined fixtures to simplify unit tests.